### PR TITLE
Test for `dashboard_utils`

### PR DESCRIPTION
**Pull request by strickvl**
2 tests for one of the `dashboard_utils` functions.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: tests/unit/utils/test_dashboard_utils.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "zenml.zen_stores.sql_zen_store.SqlZenStore.list_runs", mock_runs
+    )
+    monkeypatch.setattr(
+        "zenml.zen_stores.sql_zen_store.SqlZenStore.type", mock_store_type
+    )
+
+    if Client().zen_store.type == "rest":
+        url = dashboard_utils.get_run_url("test")
+        assert url == "https://aria_rules.com/pipelines/all-runs"
+        assert isinstance(url, str)
```
Comment (safoinme) [inline tests/unit/utils/test_dashboard_utils.py:56]: This function is meant to test get run URL, can we add another test where the runs list is not empty and the returned URL is actually right for a run name?

Comment (strickvl) [reply tests/unit/utils/test_dashboard_utils.py:56]: Added an extra test to cover this case. Thanks!

**Review by safoinme — APPROVED** [review]

**Review by htahir1 — APPROVED** [review]
